advanced/api-reference.md

```diff
@@ -2,5 +2,6 @@
 title: "API Reference"
 section: advanced
 sortOrder: 350
-callout: This section is incomplete. Please help to improve it.
```

Comment:
The API docs will at some point end up here so maybe leave this line?

Reply:
No problem, I've updated the PR.

If you want to give me a steer on what you intend for the API documentation then I could make a start on that.

Reply:
Sweet, I'll drop you a message on Slack, thank you!